### PR TITLE
Resource controllers

### PR DESCRIPTION
**Pull request by Easiv**
Generated resources, controllers and migrated the database (for the second time)

### CODE REVIEW

File: app/controllers/book_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class BookController < ApplicationController
```
Comment (Rxbsxn) [inline app/controllers/book_controller.rb]: Please take a look at the jsonapi-resources documentation. There you should find the default controller class name and you should use it here

File: app/controllers/page_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class PageController < ApplicationController
```
Comment (Rxbsxn) [inline app/controllers/page_controller.rb]: same as above

File: app/controllers/user_controller.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+class UserController < ApplicationController
```
Comment (Rxbsxn) [inline app/controllers/user_controller.rb]: same as above

File: test/controllers/book_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class BookControllerTest < ActionDispatch::IntegrationTest
```
Comment (Rxbsxn) [inline test/controllers/book_controller_test.rb]: change it for rspec

File: test/controllers/page_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PageControllerTest < ActionDispatch::IntegrationTest
```
Comment (Rxbsxn) [inline test/controllers/page_controller_test.rb]: rspec

File: test/controllers/user_controller_test.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserControllerTest < ActionDispatch::IntegrationTest
```
Comment (Rxbsxn) [inline test/controllers/user_controller_test.rb]: rspec

File: test/models/book_test.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class BookTest < ActiveSupport::TestCase
```
Comment (Rxbsxn) [inline test/models/book_test.rb]: rspec

File: test/models/page_test.rb [hunk 8]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PageTest < ActiveSupport::TestCase
```
Comment (Rxbsxn) [inline test/models/page_test.rb]: rspec

File: test/models/user_test.rb [hunk 9]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
```
Comment (Rxbsxn) [inline test/models/user_test.rb]: rspec

**Review by Rxbsxn — CHANGES_REQUESTED** [review]
Please take a look at the feedback

**Comment by Easiv** [comment]
@Rxbsxn take a look now

**Review by Rxbsxn — APPROVED** [review]

**Pull request merged**